bridge-ffi/src/lib.rs

```diff
@@ -794,6 +752,8 @@ impl tmprl_worker_t {
         self.worker.request_workflow_eviction(&req.run_id);
     }
 
+    // TODO: Fetch logs
```

Comment:
Can probably even remove this TODO, I don't think .Net is gonna use it (at least not anytime soon, if we get there, I think .Net FFI is fast enough I may request a literal per-log callback)